src/Preferences.js

```diff
@@ -73,6 +73,10 @@ define(function Preferences(require, exports, module) {
                 name: Strings.PREF_AUTOHIDE_NAME,
                 description: Strings.PREF_AUTOHIDE_DESC
             }
+        }, {
+            id: "delay",
```

Comment:
Please rename to `"autohideDelay"` (or equivalent) to indicate the connection to the autohide feature.